tests/unit/test_charm_control_plane.py

```diff
@@ -19,7 +19,6 @@ def test_install(e: Environment, is_leader: bool):
 
     e.util.install_required_packages.assert_called_once_with()
     e.microk8s.install.assert_called_once_with()
-    e.microk8s.wait_ready.assert_called_once_with()
```

Comment:
This should still be `e.microk8s.wait_ready.assert_called()`. Otherwise, someone might remove the wait for readiness checks in the future causing the regressions we already experienced, but the tests would still pass.